src/crypto/accounts.slice.ts

```diff
@@ -81,11 +89,29 @@ export function* fetchAccount(wallet: SerializedWallet): SagaIterator<Serialized
 
 export function* fetchAccountsWorker({ payload }: PayloadAction<SerializedWallet[]>) {
   try {
-    const wallets: SerializedWalletWithAddress[] = yield all(
+    const type =
```

Comment:
These two lines seem to be complicating things too much, can we abstract away?

Reply:
Maybe moving it to two separate functions? We can use a switch statement instead for the secret.

In general I just want it to be clearer that it is only used for paper wallets. Perhaps with some more comments?